### DISCUSSION TITLE
怎么在query_decode中使用vec作为查询条件？

### DISCUSSION THREAD

**Discussion by zigaziz** [rbatis/rbatis#425] [Q&A] [answered]
![image](https://github.com/rbatis/rbatis/assets/95843162/c878bbd2-1f30-4267-a3c6-a59b7e849286)
![image](https://github.com/rbatis/rbatis/assets/95843162/ab5a4708-8db5-4b9d-8358-11d66481230c)

**Comment by zhuxiujia** [comment, marked as answer]
query_decode只支持普通sql，你的需求需要用到py_sql，可以用crud!宏实现的功能，看例子https://github.com/rbatis/example/blob/main/src/crud_sql.rs
